aviary/models/propellers/PropFan.map

```diff
@@ -0,0 +1,309 @@
+    2    1
```

Comment:
Do we need to explain what these are?

Reply:
I added an explanation in `aviary_commands.ipynb`: This is the first integer on the first line of the input file.